data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsService.java

```diff
@@ -146,7 +146,7 @@ private SchemaManager getSchemaManager(final RdsSourceConfig sourceConfig, final
                 dbMetadata.getPort(),
                 sourceConfig.getAuthenticationConfig().getUsername(),
                 sourceConfig.getAuthenticationConfig().getPassword(),
-                !sourceConfig.getTlsConfig().isInsecure());
+                !sourceConfig.isTlsEnabled());
```

Comment:
Should this be `sourceConfig.isTlsEnabled()` without `!`

Reply:
Good catch. Will fix.